### PR TITLE
Fix support for critical elements

### PR DESCRIPTION
**Pull request by snorp**
- Fix support for critical elements
- Bump version to 0.2.0

### CODE REVIEW

File: mdoc-credential/src/main/java/com/google/android/mdoc/GetMdocCredentialOption.kt [hunk 1]
```diff
@@ -74,27 +76,37 @@ class GetMdocCredentialOption @JvmOverloads constructor(
...
             bundle.putByteArray(BUNDLE_KEY_NONCE, nonce)
             bundle.putSerializable(BUNDLE_KEY_PUBLIC_KEY, publicKey)
 
             bundle.putStringArrayList(
-                BUNDLE_KEY_SUPPORTED_ELEMENT_KEYS, elements
+                BUNDLE_KEY_SUPPORTED_ELEMENT_KEYS, candidateElements
+            )
+
+            // Instead of putting candidate and request elements in the candidate and request
+            // bundles, we need to send them separately due to b/294599744.
```
Comment (bslassey) [inline mdoc-credential/src/main/java/com/google/android/mdoc/GetMdocCredentialOption.kt]: this isn't a public URL

File: mdoc-credential/src/main/java/com/google/android/mdoc/GetMdocCredentialOption.kt [hunk 2]
```diff
@@ -74,27 +76,37 @@ class GetMdocCredentialOption @JvmOverloads constructor(
...
+            bundle.putStringArrayList(
+                BUNDLE_KEY_REQUESTED_ELEMENTS, requestedElements
             )
```
Comment (bslassey) [inline mdoc-credential/src/main/java/com/google/android/mdoc/GetMdocCredentialOption.kt]: Let's wrap this in a check that we're running on UDC.

**Review by bslassey — APPROVED** [review]

File: mdoc-credential/src/main/java/com/google/android/mdoc/GetMdocCredentialOption.kt (hunk 2 above)
Comment (snorp) [reply mdoc-credential/src/main/java/com/google/android/mdoc/GetMdocCredentialOption.kt]: Done

File: mdoc-credential/src/main/java/com/google/android/mdoc/GetMdocCredentialOption.kt (hunk 1 above)
Comment (snorp) [reply mdoc-credential/src/main/java/com/google/android/mdoc/GetMdocCredentialOption.kt]: Removed the reference to the bug number.

**Pull request merged**